async/src/traits/consumer.rs

```diff
@@ -75,6 +75,7 @@ impl<'a, A: AsyncConsumer> Future for PopFuture<'a, A> {
             assert!(!self.done);
             let closed = self.owner.is_closed();
             #[cfg(feature = "std")]
+            std::println!("PopFuture::poll: closed={}", closed);
```

Comment:
Don't know when I accidentally removed this, which can cause `no-std` problems. Yet `no-std` is not in the tests though.

Reply:
This accidentally causes 100% cpu infinite loop in a `pop().await` with `--no-default-features --features alloc` (not my use case though).

Reply:
I'm sorry, it was me to remove this `println!` in [subsequent commit](https://github.com/agerasev/ringbuf/commit/73be5de1d1fe096b38db8e5afd6eeb192e01d60b) after your request, but forgot to remove `#[cfg(feature = "std")]` so now it affects `match` that is wrong. This `#[cfg(...)]` needs to be removed instead of restoring `println!` back.

Reply:
Yeah. Ok. Got that removed.